internal/artifacts/module_arch_test.go

```diff
@@ -133,15 +134,22 @@ builders:
 				validateArchiveContents([]string{"test.txt", "test2.txt", "test2_copy.txt"}, getTestPath("result", "data.zip"))
 			})
 
-			It("modules m1 and m2 have conflicting build results", func() {
-				err := ExecuteSoloBuild(getTestPath("mtaModelsBuild"), getResultPath(), nil, []string{"m1", "m2"}, true, false, "", os.Getwd)
-				Ω(err).Should(HaveOccurred())
-				Ω(err.Error()).Should(ContainSubstring(fmt.Sprintf(multiBuildWithPathsConflictMsg, "m2", "m1", getResultPath(), "data.zip")))
-			})
-
 			It("fails on platform validation when mtad.yaml should be generated", func() {
 				Ω(ExecuteSoloBuild(getTestPath("mtaModelsBuild"), getResultPath(), nil, []string{"m1"}, true, true, "xx", os.Getwd)).Should(HaveOccurred())
 			})
+
+		})
+
+		It("modules m1 and m2 have conflicting build results detected on checkBuildResultsConflicts", func() {
+			err := ExecuteSoloBuild(getTestPath("mtaModelsBuild"), getResultPath(), nil, []string{"m1", "m2"}, true, false, "cf", os.Getwd)
+			Ω(err).Should(HaveOccurred())
+			Ω(err.Error()).Should(ContainSubstring(fmt.Sprintf(multiBuildWithPathsConflictMsg, "m2", "m1", getResultPath(), "data.zip")))
+		})
+
+		It("modules ui5app1 and ui5app2 have conflicting build results not detected on checkBuildResultsConflicts (patterns build results)", func() {
```

Comment:
Please add a test where the build result pattern is the same but the actual file name is different so it doesn't fail (it can be a new module in the same file)

Reply:
added